utils/config.mlp

```diff
@@ -67,22 +67,14 @@ let mkdll, mkexe, mkmaindll =
   else
     "%%MKDLL%%", "%%MKEXE%%", "%%MKMAINDLL%%"
 
-let flambda = %%FLAMBDA%%
+let default_flambda = %%FLAMBDA%%
 
 let exec_magic_number = "Caml1999X011"
 and cmi_magic_number = "Caml1999I020"
 and cmo_magic_number = "Caml1999O011"
 and cma_magic_number = "Caml1999A012"
-and cmx_magic_number =
-  if flambda then
-    "Caml1999Y016"
-  else
-    "Caml1999Y015"
-and cmxa_magic_number =
-  if flambda then
-    "Caml1999Z015"
-  else
-    "Caml1999Z014"
+and cmx_magic_number = "Caml1999Y015"
+and cmxa_magic_number = "Caml1999Z014"
```

Comment:
We probably don't want to decrease the maximum magic. Future compiler versions would risk reusing `Caml1999Y016`